allium/lib/api_diagnostics.py

```
@@ -474,6 +585,46 @@ def collect_api_diagnostics(relay_set, args):
+                    [(k, v) for k, v in v3_cats.items() if v > 0],
+                    key=lambda x: -x[1],
+                )[:5]
+                # B6 (final): warning feed from this build's aroi_validation
+                # module. Plan called for "Last warning timestamp for:
+                # unknown error_category, unknown proof_type, schema version
+                # mismatch" — surfaced here as a structured list so the
+                # template can render with timestamps.
+                try:
+                    from .aroi_validation import get_aroi_warnings_log
+                    aroi_extras["warnings"] = get_aroi_warnings_log()
+                except Exception:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)